src/pages/starter-kit/reference-app.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Reference applications
+description: Review reference applications created using the Adobe Commerce integration starter kit.
+keywords:
+ - Extensibility
+ - App Builder
+ - API Mesh
+ - Events
+ - REST
+ - Tools
+---
+
+# Reference Applications
```

Comment:
I am unsure if we should give so much importance to the fact that this is a reference application. Each reference application will have its page.  I think it is better to create that page like the MSFT F&O specific one, and in the future, if we have more than 3, think about making a page that englobe all of them.